designer/server/src/main/scala/pl/touk/nussknacker/ui/process/repository/DBIOActionRunner.scala

```diff
@@ -7,25 +7,24 @@ import slick.jdbc.JdbcProfile
 import scala.concurrent.Future
 import scala.language.higherKinds
 
-// TODO: We could rename this class to sth like DBIOActionRunner to be clear that the only responsibility of this class is to make Future[_] from DB[_]
-object RepositoryManager {
+object DBIOActionRunner {
 
-  def createDbRepositoryManager(dbConfig: DbConfig): RepositoryManager =
-    new RepositoryManager {
+  def apply(dbConfig: DbConfig): DBIOActionRunner =
+    new DBIOActionRunner {
 
       protected lazy val profile: JdbcProfile = dbConfig.driver
       protected lazy val api : profile.API = profile.api
       import api._
 
       override def runInTransaction[T](action: DB[T]): Future[T] =
-        dbConfig.run(action.transactionally)
+        run(action.transactionally)
 
       override def run[T](action: DB[T]): Future[T] =
-        dbConfig.run(action)
+        dbConfig.db.run(action)
     }
 }
 
-trait RepositoryManager {
+trait DBIOActionRunner {
```

Comment:
I'm wondering, do we need this interface right now?

Reply:
you mean that we can have a simple class instead?

Reply:
Yep, it had the sense when we had some abstractions of RepositoryManager[F], but right now we support just slick db as I understood.